setup.py

```diff
@@ -1,4 +1,4 @@
-from setuptools import setup
+from setuptools import setup, find_packages
```

Comment:
We have no custom commands, so this is not really necessary in this repository

Reply:
I already changed this but can I kindly ask you to explain to me when would this specification be necessary?

Reply:
It is only necessary in the datasets where we have custom commands defined. This is, for example, the case in `crossandean` (crossandeancommands), or the one where had discussed this initially. In such cases, the install command should also look in the local folder (`.`) to install additional commands that do not come through the generic pylexibank --> because they are stored locally in the repository

Reply:
great :) thanks for the clarification. it is much more clear when to use this or not.